batch/batch/front_end/front_end.py

```diff
@@ -411,7 +412,7 @@ async def get_completed_batches_ordered_by_completed_time(request, userdata):
         wheres.append('batches.time_completed < %s')
 
     sql = f"""
-SELECT batches.*,
+SELECT DISTINCT batches.*,
```

Comment:
All the LEFT joins are duplicating records, therefore DISTINCT

Reply:
I don't love a distinct, can we refine the left join to select only single records, or use a group by?

Reply:
OK, let me investigate what exactly is causing it and change it, it might be casued by multiple job_groups.

Reply:
I had found out the DISTINCT is not needed.
During my test 22 duplicated records where showing up because I had missed one extra condition by the user:
https://github.com/populationgenomics/hail/blob/d65f84e8988c70bcaa2e10f2d2573ad3650d5b6d/batch/batch/front_end/front_end.py#L394

And as we have 22 users in DB, it was 22x of the same batch.
So in the end DISTINCT is not needed here.